### PR TITLE
[Backport release-22.11] aws-workspaces: 4.1.0 -> 4.5.0

### PR DESCRIPTION
**Pull request by mausch**
###### Description of changes

(cherry picked from commit 695a3c582ed86e15aba60bdd6d1f64aac4aa6493) 
The previous download links were all broken

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -3822,6 +3822,18 @@
...
   };
+  d-xo = {
+    email = "hi@d-xo.org";
+    github = "d-xo";
+    githubId = 6689924;
+    name = "David Terry";
+  };
+  dylanmtaylor = {
+    email = "dylan@dylanmtaylor.com";
+    github = "dylamtaylor";
```
Comment (dylanmtaylor) [inline maintainers/maintainer-list.nix:3833]: Can you fix this to dylanmtaylor? There is a typo here.

Comment (dylanmtaylor) [inline maintainers/maintainer-list.nix:3833]:
```suggestion
    github = "dylanmtaylor";
```

**Comment by dylanmtaylor** [comment]
You can cherry-pick https://github.com/NixOS/nixpkgs/pull/225023

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (mausch) [reply maintainers/maintainer-list.nix]: cherry-picked the fix in https://github.com/NixOS/nixpkgs/pull/225163/commits/8add8664729470f2472ee91e26919f17935b327d

**Review by dylanmtaylor — APPROVED** [review]

**Comment by dylanmtaylor** [comment]
I think this is failing because the first commit to add my username also adds d-xo, which is already there.

**Comment by mweinelt** [comment]
Unfortunately NixOS 22.11 has reached its end-of-life status on 2023-06-30, one month after the release of NixOS 23.05.

I'm closing this pull request, since we do not accept any changes to its branches anymore.

**Pull request closed without merging**